parser.gemspec

```diff
@@ -59,4 +56,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov', '~> 0.15.1'
 
   spec.add_development_dependency 'gauntlet'
+  spec.add_development_dependency 'rubocop-packaging'
```

Comment:
do we need it?

Reply:
It helps to ensure that the library is easily maintainable downstream (not only Debian, but other distributions as well!).
The `Packaging` extension was explicitly created to help downstream maintainers without compromising the experience of upstream maintainers. 

Let me know what you think?

Reply:
it's a dependency that `parser` doesn't need. could you remove it please? it doesn't affect development process and it's not even required anywhere. it will only slow down CI builds for no practical reason

Reply:
fine, removed.